### PR TITLE
[feat] - Locust version update

### PR DESCRIPTION
**Pull request by jonaphael**
* **Please check if the PR fulfills these requirements**
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Upgrade version of locust tool

* **What is the current behavior?** (You can also link to an open issue here)
dojot/dojot#1494

* **What is the new behavior (if this is a feature change)?**
updated version of paho MQTT, locust tool

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
No

* **Other information**:

### CODE REVIEW

**Comment by mprevide** [comment]
Please resolve conflicts and merge the to branch feature

**Review by mprevide — APPROVED** [review]

File: connector/mqtt/locust/Docker/docker-compose-master.yml [hunk 1]
```diff
@@ -1,25 +1,25 @@
 version: '3'
 services:
   locust-master:
-    image: dojot/locust:development
+    image: jonaphael/locust:development
```
Comment (thiagodpf) [inline connector/mqtt/locust/Docker/docker-compose-master.yml:4]: > dojot/locust:development

File: connector/mqtt/locust/Docker/docker-compose-master.yml [hunk 2]
```diff
@@ -1,25 +1,25 @@
...
       # TODO: differentiate DOJOT_MQTT_PORT and DOJOT_MQTTS_PORT variables
-      DOJOT_MQTT_PORT: "30311"
+      DOJOT_MQTT_PORT: "8883"
       DOJOT_MQTT_TIMEOUT: "120"
       REDIS_HOST: "redis"
       REDIS_PORT: "6379"
       REDIS_PASSWD: ""
       REDIS_BACKUP: "y"
-      DOJOT_ENV: "y"
+      DOJOT_ENV: "n"
```
Comment (thiagodpf) [inline connector/mqtt/locust/Docker/docker-compose-master.yml:22]: > `DOJOT_ENV: "y"` ?

File: connector/mqtt/locust/Docker/docker-compose-slave.yml [hunk 3]
```diff
@@ -1,22 +1,24 @@
 version: '3'
 services:
   locust-slave:
-    image: dojot/locust:development
+    image: jonaphael/locust:development
```
Comment (thiagodpf) [inline connector/mqtt/locust/Docker/docker-compose-slave.yml:4]: > dojot/locust:development

**Review by thiagodpf — COMMENTED** [review]
Some changes are needed

**Pull request merged**